### PR TITLE
ref(ui): Make alert details chart footer same as others

### PR DESCRIPTION
**Pull request by taylangocmen**
This updates the alert details chart's footer to look more similar to discover charts'.

Before:
Screen Shot 2022-03-09 at 5 33 36 PM

After:
Screen Shot 2022-03-09 at 5 34 23 PM

Jira: [WOR-1599](https://getsentry.atlassian.net/browse/WOR-1599)

### CODE REVIEW

**Review by evanpurkhiser — APPROVED** [review]
image

my favorite kind of PR

**Comment by scttcper** [comment]
image

looks a little weird on mobile, might be spreading them

File: static/app/views/alerts/rules/details/metricChart.tsx [hunk 1]
```diff
@@ -863,47 +869,21 @@ const Filters = styled('span')`
...
+const StyledSectionValue = styled(SectionValue)`
+  display: grid;
+  grid-template-columns: repeat(3, auto);
+  gap: ${space(1.5)};
+  margin: 0 0 0 ${space(1.5)};
 `;
 
-const StatItem = styled('div')`
-  display: flex;
+const ValueItem = styled('div')`
```
Comment (scttcper) [inline static/app/views/alerts/rules/details/metricChart.tsx:879]: i think because of the number% we should add tabular nums to the items as well

Comment (taylangocmen) [reply static/app/views/alerts/rules/details/metricChart.tsx:890]: :+1:

**Pull request merged**